lib/Zonemaster/Backend/DB.pm

```diff
@@ -68,14 +68,14 @@ sub add_api_user {
 
 # Standard SQL, can be here
 sub get_test_request {
-    my ( $self ) = @_;
+    my ( $self, $queue ) = @_;
```

Comment:
Why not naming it `$lock_on_queue` directly ?

Reply:
Good point! I'll go for `$queue_label`.